src/components/RadioButtonGroup/readme.md

```diff
@@ -42,6 +42,70 @@ class SimpleRadioButtonGroup extends React.Component {
     </div>
 ```
 
+##### RadioButtonGroup brand with Border Radius
+
+```js
+import React from 'react';
+import { RadioButtonGroup } from 'react-rainbow-components';
+
+const options = [
+    { value: 'off', label: 'Off' },
+    { value: 'parking', label: 'Parking' },
+    { value: 'auto', label: 'Auto' },
+    { value: 'on', label: 'On' },
+];
+
+class BorderRadiusRadioButtonGroup extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            value: 'auto',
+        };
+        this.handleOnChange = this.handleOnChange.bind(this);
+    }
+
+    handleOnChange(event) {
+        return this.setState({ value: event.target.value });
```

Comment:
here we need different states for each RadioButtonGroup in order to not have that one controls all others